# HG changeset patch
# User Rik <rik@octave.org>
# Date 1418773147 28800
#      Tue Dec 16 15:39:07 2014 -0800
# Node ID e723e93ca2a06b958c836d121494473d41965728
# Parent  cbce5d1bcaf9429c9b36113c4669668f379d2d31
doc: Periodic spellcheck of documentation.

diff --git a/scripts/plot/util/frame2im.m b/scripts/plot/util/frame2im.m
--- a/scripts/plot/util/frame2im.m
+++ b/scripts/plot/util/frame2im.m
@@ -19,18 +19,18 @@
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {[@var{x}, @var{map}] =} frame2im (@var{f})
 ## Convert movie frame to indexed image.
 ##
 ## A movie frame is simply a struct with the fields @qcode{"cdata"} and
 ## @qcode{"colormap"}.
 ##
 ## Support for N-dimensional images or movies is given when @var{f} is a
-## struct array.  In such cases, @var{x} will be a MxNx1xK or MxNx3xK for
-## indexed and RGB movies respectively, with each frame concatenated on
+## struct array.  In such cases, @var{x} will be a @nospell{MxNx1xK or MxNx3xK}
+## for indexed and RGB movies respectively, with each frame concatenated on
 ## the 4th dimension.
 ##
 ## @seealso{im2frame}
 ## @end deftypefn
 
 ## Author: Carnë Draug <carandraug@octave.org>
 
 function [x, map] = frame2im (frame)
diff --git a/scripts/plot/util/im2frame.m b/scripts/plot/util/im2frame.m
--- a/scripts/plot/util/im2frame.m
+++ b/scripts/plot/util/im2frame.m
@@ -20,18 +20,18 @@
 ## @deftypefn  {Function File} {} im2frame (@var{rgb})
 ## @deftypefnx {Function File} {} im2frame (@var{x}, @var{map})
 ## Convert image to movie frame.
 ##
 ## A movie frame is simply a struct with the fields @qcode{"cdata"} and
 ## @qcode{"colormap"}.
 ##
 ## Support for N-dimensional images is given when each image projection,
-## matrix sizes of MxN and MxNx3 for RGB images, is concatenated on the fourth
-## dimension.  In such cases, the returned value is a struct array.
+## matrix sizes of @nospell{MxN and MxNx3} for RGB images, is concatenated on
+## the fourth dimension.  In such cases, the returned value is a struct array.
 ##
 ## @seealso{frame2im}
 ## @end deftypefn
 
 ## Author: Carnë Draug <carandraug@octave.org>
 
 function [frame] = im2frame (x, map = [])
 
